content/news/ragl-announcement.md

```diff
@@ -0,0 +1,21 @@
+<div style="text-align:center" markdown="1">
+![Global chat lobby](/images/news/20160329-ragl-announcement.png)
+</div>
+
+Greetings commanders!
+
+Our well-known community member and caster SoScared is hosting the Red Alert: Global League.
+
+Monday, April 4th 2016 will see the launch of the new Red Alert: Global League. The Global League will attempt to combine the best elements of a tough scheduled format with a flexible meet-up to circumvent most, if not all, time-zone related issues. RA:GL aims to become a long-term, easy and reliable format for anyone that wishes to participate.
+
+The League will be running for two months twice a year, hosting several divisions. As with the recent [Tag Team Tournament](/news/march-2016-tournament-announcement/), there will again be a pool of prize money to be distributed among the top contenders of the two highest divisions. If you're interested, [read the rules](http://forum.openra.net/82/19488) and [sign up](http://forum.openra.net/82/19512) on our forums. It is expected that many of the games will be casted on Twitch or Youtube, so keep an eye out for those streams if you want to follow along.
+
+We wish all participants the best of luck and a lot of fun!
+
+Meanwhile, the Tag Team Tournament was won by The AMHOLES (FrameLimiter and Barf), with a worthy 2nd place taken by Team Prize Fighters (Anjew and Amphy). Congratulations to both teams!
+
+If you have not been able to follow the tournament's live streams, fear not! Many streams have already been uploaded to Youtube, with the rest to follow in short order, where you can catch up on all games on [FiveAces'](https://www.youtube.com/user/CovertFlobert) and [Hecthor Doomhammer's](https://www.youtube.com/playlist?list=PLi5UzmvU26JbGoprGFvU0NbbPDIu17nnJ) channels.
+
+<div style="font-size:8pt; font-style:italic" markdown="1">
+\[Apologies for posting this late, but if you are following OpenRA in any way, you should have already come across this announcement on [Facebook](http://facebook.com/openra), [GameReplays.org](http://gamereplays.org/openra), directly on our forums or via the ingame battlefield news.\]
```

Comment:
I'm not convinced about the square brackets here. IMO it would look better with just the italics.  Not a blocker if you don't want to change it, though.